### ISSUE TITLE
Home page for a website

### ISSUE THREAD

**Issue by ManasiGDeshmukh** [Mozilla-Campus-Club-Cummins/Spring-of-Code#45]
Design a Home Page using HTML, CSS and JavaScript for a website with helps students improve their coding skills.
The site provides theory as well as problems to solve.

name file as index.html and index.css
home page must cards that will link pages like
(link to practice.html, theory.html and so on)
- practice
- theory
- courses
- contests
- etc.
add options as you feel suitable 
Do not include Nav Bar,

Note: The page should support multi screen dimensions

**Comment by aditisinghal1210** [comment]
Hey can u pls assign me this issue,I am done with my previous issue..
Aditi Singhal

**Comment by Rasika-Kanitkar** [comment]
I would like to work on this issue
Rasika Kanitkar SY Comp
( done with previous issue)

**Comment by ManasiGDeshmukh** [comment]
@aditisinghal1210 
Add your ans to Solution folder and create pull request
Happy Coding!

**Comment by aditisinghal1210** [comment]
Hey ! To confirm I have been assigned this right?

**Comment by ManasiGDeshmukh** [comment]
@aditisinghal1210 
It looks like your previous issue is not merged yet.
Once that is done I can assign you new issue
